tools/abel/README.md

```diff
@@ -0,0 +1,75 @@
+# ABEL
+#### Automated Building Entity Loader
+
+## Overview
+ABEL is a tool allowing system integrators to easily and efficiently create and modify [Building Configuration files](https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md).
+
+#### UDMI
+ABEL does not yet support [UDMI Site Model](https://github.com/faucetsdn/udmi/blob/master/docs/specs/site_model.md).
+
+## Setup and installation
+Similar to the [GUID Generator](https://github.com/google/digitalbuildings/tree/master/tools/guid_generator), ABEL's setup process depends on the instance validator. It is best to run the global setup script to setup the entire tools library which ABEL depends on
+
+1. `git clone https://github.com/google/digitalbuildings.git`
+2. `cd digitalBuildings/tools`
+3. Depending on your operating system run either of the global setup scripts to configure dependencies
+    * Windows: `pip_install.bat`
+    * Unix/MacOs: `sudo bash pip_install.sh`
+5. Create a directory for ABEL to use at `~/abel/` and navigate to it
```

Comment:
keeping this comment here as we discussed changing it.

Reply:
Removed from docs and removed from code as well.